python/cuml/test/test_train_test_split.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import cudf
+from cuml.preprocessing.model_selection import train_test_split
+
+
+@pytest.mark.parametrize("n_rows", [100, 1000, 10000, 100000])
```

Comment:
what do the different sizes test? It seems that `[100, 100000]` would be enough.

Reply:
At the time I didn't realize we had separate asv benchmarks, so I threw these long dataframe tests in there. I'll remove them